### PR TITLE
Location screen for fixed session moved.

### PR DESCRIPTION
**Pull request by jankrzempek**
The location screen which prompts the user to give the app access to the Location was moved to 'after SessionDetailsView' in the case when session is fixed. On the mobile it stays as it is.

https://trello.com/c/M3fRJ3wn/500-allow-users-to-create-fixed-session-even-if-they-have-location-tracking-disabled-but-keep-the-turn-on-location-view-as-a-reminde

### CODE REVIEW

File: AirCasting/CreateSessionViews/CreateSessionDetails/CreateSessionDetailsViewModel.swift [hunk 1]
```diff
@@ -64,8 +68,12 @@ class CreateSessionDetailsViewModel: ObservableObject {
     
     func compareIsIndoor(sessionContext: CreateSessionContext) -> CreateSessionContext {
         sessionContext.isIndoor = isIndoor
-        isLocationSessionDetailsActive = !isIndoor
-        isConfirmCreatingSessionActive = isIndoor
+        guard locationHandler.isLocationDenied() && !(sessionContext.isIndoor != nil) else {
```
Comment (anna1901) [inline AirCasting/CreateSessionViews/CreateSessionDetails/CreateSessionDetailsViewModel.swift:71]: `!(sessionContext.isIndoor != nil)` what do we want to check here? I think `sessionContext.isIndoor == nil` would be easier to read.  However, do we want to move forward to `isLocationScreenNedeed = true` if location isn't denied but isIndoor is nil?

File: AirCasting/CreateSessionViews/PowerABView.swift [hunk 2]
```diff
@@ -49,17 +50,9 @@ struct PowerABView: View {
...
     var continueButton: some View {
-        NavigationLink(destination: SelectPeripheralView(SDClearingRouteProcess: false, creatingSessionFlowContinues: $creatingSessionFlowContinues, urlProvider: urlProvider)) {
+        NavigationLink(destination: SelectPeripheralView(SDClearingRouteProcess: false, creatingSessionFlowContinues: $creatingSessionFlowContinues, urlProvider: urlProvider, locationHandler: locationHandler)) {
             Text(Strings.Commons.continue)
                 .frame(maxWidth: .infinity)
         }
     }
 }
-
-#if DEBUG
```
Comment (anna1901) [inline AirCasting/CreateSessionViews/PowerABView.swift:59]: Could you keep the preview please?

**Review by anna1901 — COMMENTED** [review]
I'm trying to test this by uninstalling the app and installing again and then creating fixed session, but for some reason I'm never asked about permission to track location. It is automatically granted.

File: AirCasting/CreateSessionViews/CreateSessionDetails/CreateSessionDetailsViewModel.swift (hunk 1 above)
Comment (jankrzempek) [reply AirCasting/CreateSessionViews/CreateSessionDetails/CreateSessionDetailsViewModel.swift]: We were checking that to be sure that for the first session context must not be nil before checking the whole guard statement. After that, we were ensuring ourselves about location and indoor. I changed it anyway 😄. Thanks for asking

File: AirCasting/CreateSessionViews/PowerABView.swift (hunk 2 above)
Comment (jankrzempek) [reply AirCasting/CreateSessionViews/PowerABView.swift:59]: I would like to delete it, as I talked to Paweł and after his merge of the architectural change, most of the previews will be gone from the app

**Review by anna1901 — APPROVED** [review]

File: AirCasting/CreateSessionViews/PowerABView.swift (hunk 2 above)
Comment (pawelgil) [reply AirCasting/CreateSessionViews/PowerABView.swift:59]: Previews are really annoying to maintain at this point, we should remove all of them and re-introduce using more structured approach (if necessary), sharing Mock/Dummy data providers with unit test target.

**Pull request merged**